summarizer_server/text_rank.py

```diff
@@ -10,13 +10,16 @@
 from nltk import tokenize
 from nltk import corpus
 from sklearn.metrics.pairwise import cosine_similarity
+from newspaper import Article
 from image_setup import WORD_EMBEDDINGS_FILE
+from newspaper import Article
 
 log = logging.getLogger("summarizer_server")
 
 
 class TextRank:
-    def __init__(self):
+    def __init__(self, logger=None):
+        self.logger = logger
```

Comment:
I think we can remove this and the logger parameter since we have a working logger implementation on master